projects/storefrontlib/src/cms-components/order-confirmation/components/order-confirmation-overview/order-confirmation-overview.component.ts

```diff
@@ -38,10 +38,8 @@ export class OrderConfirmationComponent implements OnInit, OnDestroy {
   }
 
   getAddressCardContent(deliveryAddress: Address): Observable<Card> {
-    return combineLatest([
-      this.translation.translate('addressCard.shipTo'),
-    ]).pipe(
-      map(([textTitle]) => {
+    return this.translation.translate('addressCard.shipTo').pipe(
+      map(textTitle => {
```

Comment:
you will be my hero if you also update similar occurrences (functions with 'return' only to shorter entry) like
```
operator(arg => ({
   prop1: prop,
   ...
  })
)
```